hitl/run.sh

```diff
@@ -31,5 +31,7 @@ python3 miniconf.py dt/sinara/dual-iir/04-91-62-d9-7e-5f afe/0='"G2"'
 python3 miniconf.py dt/sinara/dual-iir/04-91-62-d9-7e-5f afe/0='"G1"' iir_ch/0/0=\
 '{"y_min": -32767, "y_max": 32767, "y_offset": 0, "ba": [1.0, 0, 0, 0, 0]}'
 
-kill $(jobs -p)
+# Probe-run appears to sporadically fail. In that case, the job is no longer active and there's no
+# background process to kill.
+kill $(jobs -p) || true
```

Comment:
With this the job will succeed although it didn't even flash the firmware to be tested.

Reply:
What are your proposals? Otherwise, we sporadically see the `probe-run` fail to complete properly and the kill step then fails later because of it. I would propose we just use cargo-flash instead. Thoughts?

Reply:
Yeah. I don't know why it fails. I could imagine that a newer probe-rs behind probe-run might handle that st-link better.
But doing for `cargo-flash` in the meantime is fine with me. I would change that in the hitl script explicitly and not the runner `.cargo/config`.